docs/hw/display/nv3/pcrtc.rst

```diff
@@ -26,8 +26,11 @@ MMIO registers
 
 .. space:: 8 prmcio 0x1000 VGA CRTC & attribute controller registers
 
-   .. todo:: write me
+   .. todo:: complete me
 
+   0x3d0 RMA_ACCESS nv3-io-rma-access
```

Comment:
The register list should be immediately after the ..space directive to be recognized (without a blank line in between), the ..todo line should come after the reg list (and a blank line) - see bus/prma.rst for an example